02-builder-analysis/exercises/valencia-bikes.md

```diff
@@ -4,122 +4,122 @@
 * *Goal*: Getting started with CARTO BUILDER.
 * *Features Highlighted*:
   * Analysis
-* *Datasests needed*. Connect to your account the GeoJSON files provided at these pages:
+* *Datasests needed*. In this lesson we will connect the GeoJSON files provided at these pages to your account:
   - [Barrios](http://gobiernoabierto.valencia.es/en/resource/?ds=barrios&id=15befdeb-83ac-4b0f-98f6-0e17ea68ce37)
   - [Aparcabicis](http://gobiernoabierto.valencia.es/en/resource/?ds=aparcabicis&id=6c38de72-598c-4e83-a093-73b17b08bbe1)
 
 ## Contents
 
 <!-- MarkdownTOC -->
 
-- Create a map
-- Adding a virtual field to a source layer
-- Analysis 1: Filter the bike parkings inside the neighborhoods
-- Analysis 2: Intersect bikes and neighborhoods
-- Styling
-- Widgets
-- Legends and interactivity
-- Map options and publish
+1. [Create a map](#map)
```

Comment:
because this part is autogenerated by Sublime Text, I wouldn't change it or add links, the document is short enough to not need them anyway.